src/loaders/common/callback_server_wrapper.py

```diff
@@ -17,78 +17,101 @@ class Conf:
     """
     Configuration class for the workspace downloader and workspace uploader scripts.
     """
+
     def __init__(
-            self,
-            job_dir: str,
-            output_dir: str,
-            worker_function: Callable,
-            kb_base_url: str = "https://ci.kbase.us/services/",
-            token_filepath: str | None = None,
-            workers: int = 5,
-            retrieve_sample: bool = False,
-            ignore_no_sample_error: bool = False,
+        self,
+        job_dir: str,
+        output_dir: str,
+        kb_base_url: str = "https://ci.kbase.us/services/",
+        token_filepath: str | None = None,
+        au_service_ver: str = "release",
+        workers: int = 5,
+        max_task: int = 20,
+        worker_function: Callable | None = None,
+        retrieve_sample: bool = False,
+        ignore_no_sample_error: bool = False,
+        workspace_downloader: bool = True,
```

Comment:
If and when we move the download specific code to the downloader script, a number of these args should go away

Reply:
https://github.com/kbase/collections/issues/572#issuecomment-1846204024

Reply:
:+1: